test/plugin/constants.rb

```diff
@@ -23,8 +24,10 @@ module Constants
   # Attributes used for the GCE metadata service.
   PROJECT_ID = 'test-project-id'
   ZONE = 'us-central1-b'
+  ZONE2 = 'us-central1-c'
```

Comment:
Let's call these `METADATA_ZONE` and `METADATA_VM_ID`, to make it consistent with `CUSTOM_ZONE` and `CUSTOM_VM_ID` below...

Reply:
Changed.